Maestro/subscriptions.json

```diff
@@ -739,6 +745,30 @@
       ],
       "action": "core-setup-pipebuild-release-1.1.0"
     },
+    // Update dependencies in Standard master
+    {
+      "triggerPaths": [
+        "https://github.com/dotnet/versions/blob/master/build-info/dotnet/buildtools/master/Latest.txt"
+      ],
+      "action": "standard-general",
+      "delay": "00:10:00",
+      "actionArguments": {
+        "vsoBuildParameters": {
+          "ScriptFileName": "build-managed.cmd",
```

Comment:
I think this should just be `build.cmd` for standard--I see a `build-managed.sh` but no `build-managed.cmd`.